axlearn/common/input_lm.py

```diff
@@ -21,6 +21,9 @@
 # Value of "target_labels" which will be ignored in seq2seq processing.
 SEQ2SEQ_IGNORE_TARGET_LABEL = -1
 
+seed = os.environ.get("DATA_SEED")
+seed = int(seed) if seed is not None else None
+tf.random.set_seed(seed)
```

Comment:
Axlearn already sets a seed?

Reply:
I haven't seen Axlearn setting a seed anywhere, the input orders are random for every run. This PR is to fix that.

Reply:
https://github.com/apple/axlearn/blob/a15a3bcbb976c14db157a8958df368a48c614c1f/axlearn/common/input_tf_data.py#L383

Reply:
This seed is to sample from multiple data sources, I dont think this is to fix the order of samples in 1 data source.